radicle-node/src/service.rs

```diff
@@ -204,6 +207,8 @@ pub struct Service<R, A, S, G> {
     rng: Rng,
     /// Whether our local inventory no long represents what we have announced to the network.
     out_of_sync: bool,
+    /// Current tracked repository bloom filter.
```

Comment:
Maybe 'project' instead of 'repository'?  If I'm not mistaken, the storage is itself a repository with submodules.